wgpu-types/src/lib.rs

```diff
@@ -1026,6 +1026,9 @@ bitflags::bitflags! {
         ///
         /// GLES/WebGL don't support this.
         const DEPTH_TEXTURE_AND_BUFFER_COPIES = 1 << 13;
+
+        /// Supports rendering to floating-point textures.
+        const COLOR_ATTACHMENT_FLOAT = 1 << 14;
```

Comment:
I realize now, I don't think this is a scalable way to do this. Long term we want one or two sets of formats and usages for downlevel backends. In the short term though, the feature should just say "webgpu_format_features" or something and when it's not there, you need to call https://docs.rs/wgpu/latest/wgpu/struct.Adapter.html#method.get_texture_format_features to determine if you can render to that format at all. This way we can support the diverse set of texture formats that real downlevel devices support.

Reply:
In this PR what should I be doing for this?

Reply:
In this PR, lets make a feature called `WEBGPU_TEXTURE_FORMAT_SUPPORT` or something, with the description of "Supports all the texture usages described in WebGPU. If this isn't supported, you should call get_texture_format_features to get how you can use textures of a given format" and turn it on for dx11 FL11+ and off for GLES.